Automattic/pocket-casts-ios
# Fix constraints on Sleep Timer view

Fixes #

There was a duplication between safearea.bottom and bottom constraints that was causing constraint errors.

## To test

- Start the app
- Start playing an episode
- Open the player full screen
- Tap the timer option on the bottom
- Check that no constraints break alert show on the console. and all UI shows correctly
- Tap an option on the timer sheet
- Tap on timer again
- Check that no constraint is broken and all UI shows correctly

## Checklist
